libs/api/domains/vehicles/src/models/getVehicleDetail.model.ts

```diff
@@ -0,0 +1,256 @@
+import { Field, ObjectType } from '@nestjs/graphql'
```

Comment:
This goes for the whole file. The names are too broad and are not bound to the domain.
This means that this could be overwriting existing types in the graphql schema when it merges.
The names need to be prefixed with the domain name `Vehicles`

Reply:
I think I fixed it 😇 🤔